lib/docker-compose-definitions-splunk-helpers.yml

```diff
@@ -0,0 +1,19 @@
+x-tx-gen-def:
+  &tx-gen-def
+  image: splunkdlt/ethereum-transaction-generator:latest
+  entrypoint:
+    - tail
+    - -f
+    - /dev/null
```

Comment:
since we're not using the image's default entrypoint anyway, we can replace this with what you currently have at the end of the start script:
```
  entrypoint:
    - /bin/sh
    - -c
    - |
      echo "Waiting for nodes to come up before starting txn gen..."
      sleep 60
      echo "Starting transaction generator..."
      cd /txgen
      ./start.sh
```

This should work exactly the same way as you currently have it, with the benefit of it running in the background instead of taking over the console as it is right now.

Reply:
this was removed